compliance_checker/tests/test_cf.py

```diff
@@ -2449,6 +2452,59 @@ def test_taxonomy_skip_lsid(self):
         assert len(results[0].msgs) == 0
         assert results[0].value[0] == results[0].value[1]
 
+        
+class TestCF1_9(BaseTestCase):
+    def setUp(self):
+        self.cf = CF1_9Check()
+
+    def test_check_data_types(self):
+        dataset = MockTimeSeries()
+        dataset.createVariable("ubyte", "u1", ())
+        dataset.createVariable("ushort", "u2", ())
+        dataset.createVariable("uint", "u4", ())
+        dataset.createVariable("int64", "i8", ())
+        dataset.createVariable("uint64", "u8", ())
+        result = self.cf.check_data_types(dataset)
+        assert result.value[0] == result.value[1]
```

Comment:
TODO: Add failing case.  Since all the simple types supported by netCDF4 module should be handled, I think we'd have to create a compound type.  See https://unidata.github.io/netcdf4-python/#Dataset.createVariable for more details.